feconf.py

```diff
@@ -33,6 +33,10 @@
 IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
 
 # Whether we should serve the development or production experience.
+# This is environment specific variable(DEV_MODE) changes to true only
```

Comment:
when server starts there are variable that are set based on the environment the server has been started from, and DEV_MODE is set to true only if the SERVER_SOFTWARE is development meaning is just run from the local computer.